src/App.css

```diff
@@ -630,6 +630,15 @@ This prevents table cols being rendered as multiline content, which causes inaac
   min-height: 1.9em;
 }
 
+/** Assign annotation height on single line truncated url */
+.single-line .thought {
+  /**
+    .singl-line element has display: inline-block property, it affects the height calculation of the .thougt div so we need to set the height manually.
+    This height 1.9em is the same value as the min-height above: line: 630
```

Comment:
We should avoid specifying exact line numbers in comments, as they are quickly invalidated if any code is added or removed above.